model/modbus_query.py

```
@@ -57,11 +57,9 @@ def read_registers(self, address: int, count: int, unit: int = 100) -> list:
         """
         try:
             response = self.client.read_holding_registers(address=address, count=count, unit=unit)
-            response = response.registers
+            return response.registers
         except (ModbusIOException, ModbusException, ConnectionException, AttributeError) as exe:
```

Ruff (0.7.0)

Local variable `exe` is assigned to but never used

Remove assignment to unused variable `exe`

(F841)